### ISSUE TITLE
about license

### ISSUE THREAD

**Issue by nicosomb** [wallabag/wallabag#179]
What do you think about the license of poche: WTFPL (https://github.com/inthepoche/poche/blob/master/COPYING)? 
Is that too permissive? 
Should we change the license?

**Comment by raf64flo** [comment]
Since some used licenses (j0k3r/graby) are under AGPL, wouldn't Wallabag under AGPL too? Or perhaps @j0k3r allowed to integrate its lib under MIT?

**Comment by j0k3r** [comment]
graby is a fork of [full-text-rss](https://bitbucket.org/fivefilters/full-text-rss/src/b57a2b7eedf3e507ffe30a39e122e3f0dafccb59/license.txt?at=master&fileviewer=file-view-default) which is AGPL. That's why I used AGPL too.

We already discussed about [that](https://gitter.im/wallabag/wallabag?at=58cac15c7b3f37e75403fac2) and since I didn't rewrite enough stuff to be really different from the fork I cannot change the license.

I think we finished by saying that we should ask @fivefilters if I can change the license of graby from AGPL to MIT.
